setup.py

```diff
@@ -80,9 +80,9 @@
         'Programming Language :: Python :: 2',
         'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.3',
-        'Programming Language :: Python :: 3.4',
```

Comment:
Please remove 3.3 and 3.4 also from `.travis.yml` and `tox.ini`, so that we are consistent everywhere.  (This may also apply to your other PRs...)